test/lit/passes/optimize-instructions-bulk-memory.wast

```diff
@@ -0,0 +1,162 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; NOTE: Assertions have been generated by update_lit_checks.py
```

Comment:
I'm confused by these messages.  The first appears twice. And then there is one without "and should not be edited" - are some files meant to be edited?

Reply:
Oops! There was previously a bug in the script where repeated invocations would add duplicates of this message (which also changed to say "and should not be edited" in the latest update. I got sidetracked half way through going through the file and deduplicating the message. I'll clean them up now.